event.c

```diff
@@ -249,7 +249,19 @@ int EnableChannel(EVT_HANDLE hChannel, int status) {
     if  (!EvtSetChannelConfigProperty((EVT_HANDLE)hChannel, EvtChannelConfigEnabled, 0, &ChannelProperty)) {
        return 1;
     }
+    return 0;
+}
+
+ULONGLONG GetBufferSizeB(EVT_HANDLE hChannel, ULONGLONG* res) {
+    DWORD dwBufferSize = 16; // EvtChannelLoggingConfigMaxSize property size
```

Comment:
Looks a bit scary. I think we could afford to make 2 calls: first identifying a buffer size and second actually grabbing the data